test/unit/core.js

```diff
@@ -1190,6 +1190,18 @@ test("jQuery.each(Object,Function)", function() {
 	equal( i, document.styleSheets.length, "Iteration over document.styleSheets" );
 });
 
+test("jQuery.each/map(undefined/null,Function)", 1, function() {
```

Comment:
Oh the humanity! How can we possible recover from this horrible-horrible mistake?! Sorry, could resist. 

We will fix all of that occurrences at some point.